lib/Styles/Core.less

```diff
@@ -204,11 +204,7 @@ select {
 
   p {
     margin-top: 0;
-    margin-bottom: 1rem;
-  }
-
-  p:last-child {
-    margin-bottom: 0;
+    margin-bottom: 0; // 1rem;
```

Comment:
yep - because markdown put everything in <p> tags, but linkify doesn't.  So we can go back to the normal <p> styling